atmo/views.py

```diff
@@ -35,28 +36,50 @@ def dashboard(request):
         with_superuser=False,
     )
 
-    spark_jobs = get_objects_for_user(
+    sparkjob_qs = SparkJob.objects.all().order_by('-start_date')
+
+    user_spark_jobs = get_objects_for_user(
         request.user,
         'jobs.view_sparkjob',
-        SparkJob.objects.all().order_by('-start_date'),
+        sparkjob_qs,
         use_groups=False,
         with_superuser=False,
     )
 
-    # a list of modification datetimes of the clusters and Spark jobs to use for getting the
-    # last changes on the dashboard
+    try:
+        group = Group.objects.get(name='Spark job maintainers')
+    except Group.DoesNotExist:
+        group = None
```

Comment:
Yeah, no problem with try/except blocks in general, I prefer them as well, but this is such a common case it was added to Django a while ago.